### PR TITLE
Pull-request Title

### PR DESCRIPTION
**Pull request by LefterisGO**
added LLLLL

### CODE REVIEW

File: README.md-edits [hunk 1]
```diff
@@ -1,5 +1,5 @@
 e<header>
-LLLLLLLLLLLL
+KKKKKKKKKKKK
```
Comment (LefterisGO) [inline README.md-edits:2]: lalalala

File: README.md-edits [hunk 2]
```diff
@@ -1,5 +1,5 @@
 e<header>
-
+KKKKKKKKKKKK
```
Comment (gabbyatanasova) [inline README.md-edits:2]: I think this is better

File: README.md-edits [hunk 3]
```diff
@@ -1,5 +1,5 @@
 e<header>
-
+LLLLLLLLLLLL
```
Comment (gabbyatanasova) [inline README.md-edits]: !

Comment (gabbyatanasova) [reply README.md-edits]: h

File: README.md-edits (hunk 1 above)
Comment (gabbyatanasova) [reply README.md-edits:2]: [Moderators.docx](https://github.com/LefterisGO/skills-introduction-to-githhub/files/12660962/Moderators.docx)
 mod

**Pull request merged**